modules/gapi/test/common/gapi_imgproc_tests.hpp

```diff
@@ -50,6 +50,11 @@ GAPI_TEST_FIXTURE(SobelTest, initMatrixRandN, FIXTURE_API(CompareMats,int,int,in
     cmpF, kernSize, dx, dy)
 GAPI_TEST_FIXTURE(SobelXYTest, initMatrixRandN, FIXTURE_API(CompareMats,int,int,int,int), 5,
     cmpF, kernSize, order, border_type, border_val)
+GAPI_TEST_FIXTURE(LaplacianTest, initMatrixRandN, FIXTURE_API(CompareMats,int),
+                  2, cmpF, kernSize)
```

Comment:
It would be better to use the same variable names everywhere: the G-API operation wrapper, kernel's run method and every other place where you expect variables reflecting the same entities (e.g: test parameters). This will give user opportunity to perceive the code in different places more easily. (I mean, e.g., kernSize -> kSize here)

Reply:
Other tests contain the kernSize parameter, and their functions get kSize. I prefer standardization.

Reply:
I see. OK